apps/src/templates/sectionsRefresh/SectionsSetUpContainer.jsx

```diff
@@ -111,6 +127,51 @@ export default function SectionsSetUpContainer({sectionToBeEdited}) {
     setAdvancedSettingsOpen(!advancedSettingsOpen);
   };
 
+  const recordSectionSetupEvent = (e, section) => {
+    e.preventDefault();
+    const initial = initialSection.current;
+    // We do not currently store version year on the section, and the version dropdown
+    // does not update it. We will need to query all course offerings or set up a new
+    // course offerings controller function in order to populate these values.
+    let versionYear;
+    let initialVersionYear;
```

Comment:
Are these used?

Reply:
They're not, but I reference them in the comment above. Happy to just remove them and make that comment more explicit about what I'm referring to?

Reply:
Okay I updated the comment to be more explicit and made the logging send 'null'- I wanted to keep them in there so we know those are the fields we eventually want to add!